app/src/main/java/com/kabouzeid/gramophone/ui/activities/AlbumDetailActivity.java

```diff
@@ -327,6 +388,7 @@ public void setStatusbarColor(int color) {
     private void setAlbum(Album album) {
         this.album = album;
         loadAlbumCover();
+        loadWiki();
```

Comment:
It'd be preferable for the data to be fetched if the user explicitely opens the wiki dialog, to prevent unnecessary data usage.